packages/next/build/webpack-config.ts

```diff
@@ -1626,7 +1626,7 @@ export default async function getBaseWebpackConfig(
             {
               ...codeCondition,
               issuerLayer: WEBPACK_LAYERS.middleware,
-              use: getBabelOrSwcLoader(),
+              use: defaultLoaders.babel,
```

Comment:
Why do we need this change?

Reply:
it's the same value 🤔 just align with other places, so you can tweak the loader in one place